src/EmployeeBook.java

```diff
@@ -0,0 +1,191 @@
+                System.out.println(employee.getFullName() + " / " + employee.getDepartment() + " / " + employee.getSalary());
+            }
+        }
+    }
+
+    public void findSalary(int salary) {
+        for (int i = 0; i < countId; i++) {
+            Employee employee = employees[i];
+            if (employee.getSalary() == salary) {
+                System.out.println(employee.getFullName() + " / " + employee.getDepartment() + " / " + salary);
+            }
+        }
+    }
+
+    public int findMaxSalary() {
+        int maxSalary = employees[0].getSalary();
+        for (int i = 0; i < countId; i++) {
+            Employee employee = employees[i];
+            if (employee.getSalary() >= maxSalary) {
+                maxSalary = employee.getSalary();
+            }
+        }
+        return maxSalary;
+    }
+
+    public int findMinSalary() {
+        int minSalary = employees[0].getSalary();
+        for (int i = 0; i < countId; i++) {
+            Employee employee = employees[i];
+            if (employee.getSalary() <= minSalary) {
+                minSalary = employee.getSalary();
+            }
+        }
+        return minSalary;
+    }
+
+    public int amountOfCosts() {
+        int summ = 0;
+        for (int i = 0; i < countId; i++) {
+            Employee employee = employees[i];
+            summ += employee.getSalary();
+        }
+        return summ;
+    }
+
+    public double averageSalary() {
+        return (double) amountOfCosts() / employees.length;
+    }
+
+    public int getCountId() {
+        return countId;
+    }
+
+    public void salaryIndexation(int indexationPercentage) {
+        double percentage = 1 + ((double) indexationPercentage / 100);
+        for (int i = 0; i < countId; i++) {
+            Employee employee = employees[i];
+            employee.setSalary((int) (employee.getSalary() * percentage));
+        }
+    }
+
+    public void findDepartmentEmployees(int department) {
+        for (int i = 0; i < countId; i++) {
+            Employee employee = employees[i];
+            if (employee.getDepartment() == department) {
+                System.out.println(employee.getFullName() + " / " + employee.getSalary());
+            }
+        }
+    }
+
+    public void findMaxSalaryInDepartment(int department) {
+        int max = 0;
+        for (int i = 0; i < countId; i++) {
+            Employee employee = employees[i];
+            if (employee.getDepartment() == department) {
+                if (employee.getSalary() >= max) {
+                    max = employee.getSalary();
+                }
+            }
+        }
+        findSalary(max);
+    }
+
+    public void findMinSalaryInDepartment(int department) {
+        int min = employees[0].getSalary();
+        for (int i = 0; i < countId; i++) {
+            Employee employee = employees[i];
+            if (employee.getDepartment() == department) {
+                if (employee.getSalary() <= min) {
```

Comment:
```suggestion
            if (employee.getDepartment() == department && employee.getSalary() <= min) {
```
лучше избавляться от вложенности